x-pack/plugins/security_solution/common/endpoint/schema/trusted_apps.ts

```diff
@@ -30,10 +30,16 @@ export const GetTrustedAppsRequestSchema = {
 };
 
 const ConditionEntryTypeSchema = schema.literal('match');
-const ConditionEntryOperatorSchema = schema.oneOf([
-  schema.literal(OperatorEntryField.included),
-  schema.literal(OperatorEntryField.wildcard_caseless),
-]);
+// when field === PATH -> operator in ('included', 'wildcard_caseless') else operator === 'included'
+const ConditionEntryOperatorSchema = schema.conditional(
```

Comment:
Could you add some test cases for this?

Reply:
Good idea. I'll do that

Reply:
done in baac4522ecc069a02fde022a92c484e1ca4acc5e